launchable/test_runners/pytest.py

```diff
@@ -137,14 +132,41 @@ def _pytest_formatter(test_path):
 def record_tests(client, json_report, source_roots):
 
     def data_builder(case: TestCase):
+        props = case.child(Properties)
+        result = {}
+        if props is not None:
+            markers = []
+            marker: Dict[str, str] = {}
+            for prop in props:
+                if prop.name in marker:
+                    """
+                        Here is an example of an XML file.
+                        As you can see, all property tags are included within a single properties tag,
+                        so we identify each marker object by checking if a name attribute with the same value exists.
+                        ```
+                            <properties>
+                                <property name="name" value="parametrize" />
+                                <property name="args" value="('y', [2, 3])" />
+                                <property name="kwargs" value="{}" />
+                                <property name="name" value="parametrize" />
+                                <property name="args" value="('x', [0, 1])" />
+                                <property name="kwargs" value="{}" />
+                            </properties>
+                        ```
+                    """
+                    markers.append(marker.copy())
```

Comment:
I'm not sure best marker's data format. But it seems this logic depends on the order of properties. 
If grouping is not important, we can adopt test path like data format.

```
[
  {"type": "name", "value": "parametrize"},
  {"type": "arg", "value:" ('y', [2,3])"},
  {"type": "kwargs", "value", "{}"
   ...
]
```

WDTY??

Reply:
Okay, let me fix it based on your proposal.